### ISSUE TITLE
An argument named "call_module_type" is not expected here. (update docs?

### ISSUE THREAD

**Issue by davidsielertpge** [terraform-linters/tflint#1939]
### Summary

Using the configuration given in the documentation @ https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/config.md results in an error

### Command

tflint

### Terraform Configuration

```terraform
not required
```

### TFLint Configuration

```HCL
config {
  format = "compact"
  plugin_dir = "~/.tflint.d/plugins"

  call_module_type = "local"
  force = false
  disabled_by_default = false

  ignore_module = {
    "terraform-aws-modules/vpc/aws"            = true
    "terraform-aws-modules/security-group/aws" = true
  }

  varfile = ["example1.tfvars", "example2.tfvars"]
  variables = ["foo=bar", "bar=[\"baz\"]"]
}

plugin "aws" {
  enabled = true
  version = "0.4.0"
  source  = "github.com/terraform-linters/tflint-ruleset-aws"
}

rule "aws_instance_invalid_type" {
  enabled = false
}
```

### Output

```Shell
Failed to load TFLint config; C:\Users\xxxx\.tflint.hcl:7,3-19: Unsupported argument; An argument named "call_module_type" is not expected here.:

Error: Unsupported argument

  on C:\Users\dq1f\.tflint.hcl line 7:
  (source code not available)

An argument named "call_module_type" is not expected here.
```

### TFLint Version

0.49.0

### Terraform Version

1.6.5

### Operating System

- [ ] Linux
- [ ] macOS
- [X] Windows

**Comment by bendrucker** [comment]
Duplicate of https://github.com/terraform-linters/tflint/issues/1935
